slice.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Creates a subarray which starts at optional second argument and ends before the optional third argument.
+ * @param {array} array - The array to be sent into the function.
+ * @param {int} start - The index where to start the slice.
+ * @param {int} end - The index where to end the slice.
+ * @returns
+ */
+
+export default function slice(array, start, end) {
```

Comment:
Please stop naming your parameters/variables after their data type. Use more semantic naming such as `toSlice` or `targetArr`

Reply:
changed parameter name to `toSlice`